tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py

```diff
@@ -130,12 +131,17 @@ def start_agent(request, get_configuration):
     truncate_file(CLIENT_KEYS_PATH)
     time.sleep(1)
 
-    control_service('stop')
-    agent_auth_pat = 'bin' if platform.system() == 'Linux' else ''
-    call([f'{WAZUH_PATH}/{agent_auth_pat}/agent-auth', '-m', SERVER_ADDRESS])
-    control_service('start')
+    try: 
+        control_service('stop')
+        agent_auth_pat = 'bin' if platform.system() == 'Linux' else ''
+        call([f'{WAZUH_PATH}/{agent_auth_pat}/agent-auth', '-m', SERVER_ADDRESS])
+        control_service('start')
 
-    yield
+    except Exception:
+        print("Expected exception occurred")
```

Comment:
Is it expected?

Reply:
[fe11e5f](https://github.com/wazuh/wazuh-qa/pull/4205/commits/fe11e5ffe82b1ff89533d450a0a2a0745ff8e1cc)
Message modified for enhanced clarity.